CONTRIBUTING.md

```diff
@@ -8,12 +8,36 @@ Community contributions rock and we are psyched you're reading this document!
 - Feature requests are welcome and go [here](https://github.com/stjude-rust-labs/wdl/discussions/categories/feature-requests)
 - Lint rule proposals go [here](https://github.com/stjude-rust-labs/wdl/discussions/categories/rule-proposals)
 
-## FAQs
+## How can I start contributing?
```

Comment:
This is still phrased as a question, which suggests to me it can probably stay under the `FAQs` header

Reply:
To me, it seems like the information on how to contribute is the main point of the document. Then we have a list of frequent questions related to contribution. I suppose we could remove the question phrasing of this section.

Reply:
True. I'm fine with that. I think the question phrasing is ok to have outside FAQs